### ISSUE TITLE
Low recall on training data with 2.5 training loss on VOC

### ISSUE THREAD

**Issue by lovekesh-thakur** [pjreddie/darknet#775]
I am training tiny-yolo version 3 on voc dataset(2007 - train+val, 2012-train) using cfg from pjreddie website for coco. My training loss after 30000 iterations (batch size 64) is hovering around 2.5. 
But when i use validate_detector_recall to get recall, i am getting NaNs in IOUs and 40% recall for 2007 val. 
I don't understand how can i get so less recall for the data i am training it on. 
Please help.
